docs/README.md

```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
-| `phoneSelector` | `string` | CSS Selector that will wrap all the phone numbers displayed on the UI. | `undefined` |
+|    Prop name    |   Type   |                                                          Description                                                           | Default value |
+| :-------------: | :------: | :----------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|     `title`     | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
+| `imageSelector` | `string` |                                CSS Selector that will wrap all the images displayed on the UI.                                 |  `undefined`  |
+| `phoneSelector` | `string` |                             CSS Selector that will wrap all the phone numbers displayed on the UI.                             |  `undefined`  |
 
-> ⚠️ *Both `imageSelector` and `phoneSelector` must be declared with valid values in order to provide Structured Data for SEO purposes.*
+> ⚠️ _Both `imageSelector` and `phoneSelector` must be declared with valid values in order to provide Structured Data for SEO purposes._
 
 #### `store-name` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `text` | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
-| `tag` | `string` | HTML element to wrap the `store-name` block when rendered on the UI. | `div` |
+| Prop name |   Type   |                                                          Description                                                           | Default value |
+| :-------: | :------: | :----------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+|  `text`   | `string` | Text to be displayed alongside with the store name. Use `{storeName}` as value to display the store's accurate name on the UI. | `{storeName}` |
+|   `tag`   | `string` |                              HTML element to wrap the `store-name` block when rendered on the UI.                              |     `div`     |
 
 #### `store-back-link` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
+| Prop name |   Type   |                          Description                          |    Default value     |
+| :-------: | :------: | :-----------------------------------------------------------: | :------------------: |
+|  `label`  | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
```

Comment:
No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!